### DISCUSSION TITLE
Wiki Page Improvement Ideas

### DISCUSSION THREAD

**Discussion by excalith** [excalith/excalith-start-page#32] [Ideas]
Since this project started to gain attention, I created a Wiki page to help people easily understand and deploy their own start page. However, I would like to hear feedback from you regarding the information on the Wiki page, so I can further improve the pages.

Some questions on my mind are;
- Are the categories make sense?
- Is getting started section is informative and simple enough for you?
- Do you think I should add something? Or Improve a spesific topic further?

Any kind of feedback is welcome!

**Comment by LevYas** [comment]
Merhaba! Overall it's very good. 👍
I want to host the startpage locally without Docker. I followed your instructions on how to set up a fork. One thing that can be improved [at](https://github.com/excalith/excalith-start-page/wiki/Fork#serve):

>Use `yarn export` command to export the project into out folder.

This command produces the error
>The "next export" command has been removed in favor of "output: export"

But the output is already set to "standalone". Unfortunately, I wasn't able to run the project in production mode, I'll create an issue about it.

**Reply by excalith** [reply]
Merhaba mate! I am really glad that you liked this project! I just realised I forgot to update the redundant `yarn export` command. Now that we support docker and different stuff with it, I had to use API calls. So, exporting statically is now impossible. However, it is still possible to build and host it locally, just like `yarn dev` command. Will update the documentation for possible third party solutions to hosting locally after some digging.

Thanks for bringing that up though, refactored the code and the documentation but really missed the export part.
